test/agentchat/contrib/test_web_surfer.py

```diff
@@ -0,0 +1,168 @@
+import os
+import sys
+import re
+import pytest
+from autogen import ConversableAgent, UserProxyAgent, config_list_from_json
+from autogen.oai.openai_utils import filter_config
+from conftest import skip_openai
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+from test_assistant_agent import KEY_LOC, OAI_CONFIG_LIST  # noqa: E402
+
+BLOG_POST_URL = "https://microsoft.github.io/autogen/blog/2023/04/21/LLM-tuning-math"
+BLOG_POST_TITLE = "Does Model and Inference Parameter Matter in LLM Applications? - A Case Study for MATH | AutoGen"
+BING_QUERY = "Microsoft"
+
+try:
+    from autogen.agentchat.contrib.web_surfer import WebSurferAgent
+except ImportError:
+    skip_all = True
+else:
+    skip_all = False
+
+try:
+    from openai import OpenAI
+except ImportError:
+    skip_openai = True  # noqa: F811, set skip_openai to True since it's not installed.
```

Comment:
The way we skip openai tests has changed. Please check other tests under contrib/

Reply:
Done.